src/Item/Item.router.ts

```diff
@@ -108,13 +108,14 @@ export class ItemRouter extends Router {
     return new Item(attributes).upsert()
   }
 
-  uploadItemFiles = async (req: Request) => {
+  uploadItemFiles = async (req: Request, res: Response) => {
     const id = server.extractFromReq(req, 'id')
     try {
-      const uploader = getFileUploader({ acl: ACL.publicRead }, (_, file) =>
-        new S3Item(id).getFileKey(file.fieldname)
-      )
-      return utils.promisify<boolean>(uploader.any())
+      const uploader = getFileUploader({ acl: ACL.publicRead }, (_, file) => {
+        return new S3Item(id).getFileKey(file.fieldname)
+      })
+      const handler = utils.promisify<boolean>(uploader.any())
+      await handler(req, res)
```

Comment:
up to you, but we could copy the way we do this on Asset.router by caching this handler before using it on `mount()`
It's effectively the same, but just to keep the same implementation